packages/babylon/package.json

```diff
@@ -24,12 +24,6 @@
   },
   "devDependencies": {
     "@babel/helper-fixtures": "7.0.0-beta.38",
-    "babel-plugin-transform-charcodes": "0.0.10",
```

Comment:
cant we have those (`charcodes`) in 2 places and let lerna link them? I mean - it only makes sense to me to have them here, it will be super weird to see uknown dep in the source files and not see it in package.json at all

Reply:
That would simplify the process but the usage is very specific to Babylon and char codes.